resources/views/ingame/layouts/main.blade.php

```diff
@@ -1032,6 +1040,14 @@ function initChatAsync() {
                                         <span class="planet-koords ">[{!! $planet->getPlanetCoordinates()->asString() !!}]</span>
                                     </a>
                                     <!--
+                                        @if (!empty($build_active) && $planet->getPlanetId() == $currentPlanet->getPlanetId())
+                                            <a class="constructionIcon tooltip js_hideTipOnMobile tpd-hideOnClickOutside"
+                                                data-link="{{ $urlToCurrentWithUpdatedParam }}"
+                                                href="{{ $urlToCurrentWithUpdatedParam }}" title=""
+                                                previewlistener="true">
+                                                <span class="icon12px icon_wrench"></span>
+                                            </a>
+                                        @endif
```

Comment:
I see that you added a change for the constructionIcon but it's commented out now. In order to make this work properly I think the best way is to add a new method to the PlanetService class which checks if something is under construction for that planet. Then this new method can be called during the PlanetList render.